src/modules/solution/dtos/create-solution-hashtags-request-body.dto.ts

```diff
@@ -0,0 +1,32 @@
+import { isNil } from '@nestjs/common/utils/shared.utils';
+import { ApiProperty, PartialType, PickType } from '@nestjs/swagger';
+import { Lesson, LessonSolution } from '@prisma/client';
+import { SOLUTION_HASHTAG_LENGTH } from '@src/constants/constant';
+import { IsRecord } from '@src/decorators/is-record.decorator';
+import {
+  ArrayMaxSize,
+  ArrayNotEmpty,
+  IsArray,
+  IsString,
+  Length,
+} from 'class-validator';
+
+export class CreateSolutionHashtagsRequestBodyDto {
+  @ApiProperty({
+    description:
+      'string 형식으로 보내주세요. 각 항목별 최대 길이는 15자입니다. ',
+    example: ['개발', 'Web', 'React'],
+    minimum: 1,
+    maximum: 5,
+    uniqueItems: true,
+    type: () => String,
```

Comment:
```suggestion
    type: () => [String],
```
또는
```suggestion
    type: () => String,
    isArray: true,
```
로 되어야하지 않나요??
추가로 maxLength 였나 하는 길이에 대한 정보를 주는 필드도 존재합니다

Reply:
저는 maximum을 그런 이유에 사용한거였습니다. 애매하군요